types/benchmark_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
Benchmark can just use the manually value instead of rand value, so we can remove the dependency of rtutil package.

Reply:
I will remove the dependency :)